ingest-app/src/cmr/ingest/services/ingest_service/subscription.clj

```diff
@@ -25,11 +29,48 @@
   [context concept]
   (let [metadata (:metadata concept)
         subscription (spec/parse-metadata context :subscription (:format concept) metadata)
-        concept (add-extra-fields-for-subscription context concept subscription)
+        concept (->> concept
+                     (add-extra-fields-for-subscription context subscription)
+                     (subscriptions/set-subscription-arn-if-applicable context :subscription))
         {:keys [concept-id revision-id]} (mdb2/save-concept
                                           context
                                           (assoc concept :provider-id (:provider-id concept)
                                                  :native-id (:native-id concept)))]
+    (subscriptions/add-or-delete-ingest-subscription-in-cache context concept)
     {:concept-id concept-id
      :native-id (:native-id concept)
      :revision-id revision-id}))
+
+(declare delete-subscription)
```

Comment:
We should only need to declare something if it's used in the file prior to being called but I see it is defined immediately afterwards - is this some issue with the `defn-timed` macro?

Reply:
There were thousands of linter errors, so we had to conduct a campaign to remove them sometime last year. This is one of the outcomes of that effort.